### DISCUSSION TITLE
Huoltoikkuna 11.10.

### DISCUSSION THREAD

**Discussion by CodoDerDritte** [KohaSuomi/Koha#821] [Huolto- ja häiriötiedotteet]
11.10. Huoltoikkunassa Koha-installaatioiden ajot ryhmitellään uudelleen siten että kaksi edustapalvelinta ajavat tuotantoinstallaatioita ja kaksi ajavat testejä, nextejä ym.

Tunneloituja automaatteja käyttävät (tuotanto) Koha-installaatiot ryhmitellään tässä vaiheessa Amy-palvelimelle (node 4). Tunneloitujen automaattien yhteysosoite Vaara, Vaski ja Helle-kirjastoissa on keskiviikosta alkaen **10.0.36.47**. Vaskissa tämä ei aiheuta muutoksia, Hellessä ja Vaarassa IP-osoite automaateille täytyy vaihtaa keskiviikkoaamuna.

Muutokset aiheuttavat tässä vaiheessa käyttökatkon kimppojen tuotannoissa.

**Comment by CodoDerDritte** [comment]
Myös tiedostojärjestelmän asetuksiin tehdään muutoksia. Näiden pitäisi parantaa järjestelmän kirjoitussuorituskykyä, mutta muuten olla kirjastojen kannalta näkymättömiä.

**Comment by CodoDerDritte** [comment]
Huoltoa silmälläpitäen otamme testikohat alas huomenna (10.10.) iltapäivällä.

**Comment by CodoDerDritte** [comment]
Huolto alkaa, käytetään ensin alhaalla Kirkes, Kyyti ja Vaara.

**Comment by CodoDerDritte** [comment]
Outi siirtyy nyt uuteen kotiinsa.

**Comment by CodoDerDritte** [comment]
Sitten Lumme.

**Comment by CodoDerDritte** [comment]
Sitten Siili.

**Comment by CodoDerDritte** [comment]
ja lopuksi Kirkes ja Kyyti. Nuo edellämainitut on siis nyt uudessa kodissaan ja Kirkeksen, Kyytin, Lumpeen, Outin ja Siilin osalta katko päättyy tähän.

**Reply by pknuuti** [reply]
Mikä on Vaaran tilanne?

**Comment by CodoDerDritte** [comment]
Seuraavaksi pysähtyvät Lappi ja Vaski.

**Comment by CodoDerDritte** [comment]
> Mikä on Vaaran tilanne?

Odottaa uuden kotinodensa boottia, se menee samalle nodelle kuin Vaski ja Helle, jotta tunneloitujen automaattien IP on sitten kaikissa sama.

**Comment by pknuuti** [comment]
Vaara näyttäis olevan käynnissä.

**Comment by CodoDerDritte** [comment]
Lappi, Helle, Vaara, Vaski ja Täti nyt jälleen käynnissä ja huoltokatko tuotantojen osalta päättyy tähän.
